experiments/bert/runner.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from pathlib import Path
+
+from experiments.bert.bert import *
+from transfer_nlp.plugins.config import ExperimentConfig
+
+logger = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+    home_env = str(Path.home() / 'work/transfer-nlp-data')
+
+    path = './bert.json'
+    experiment = ExperimentConfig.from_json(path, HOME=home_env)
+    # Prepare optimizer
+
+    layers = ['classifier.weight', 'classifier.bias']
+
+    [parameter.requires_grad_(False) for name, parameter in experiment['trainer'].model.named_parameters() if name not in layers]
+    param_optimizer = list(experiment['trainer'].model.named_parameters())
+    optimizer_grouped_parameters = [
```

Comment:
is this a rough edge with the framework? what should be ideal here?

Reply:
I think we need to include back the freeze/unfreeze methods in the trainer class